.github/workflows/generate_issues.yml

```diff
@@ -21,3 +22,19 @@ jobs:
         access-token: '${{ secrets.GITHUB_TOKEN }}'
         # This is to exclude workflows that will be searched in the nightly report.
         exclude-workflow-files: 'performance-integration-tests.yml'
+
+  test_generate_an_issue:
+    # Don't run on private repo.
+    if: github.repository == 'Firebase/firebase-ios-sdk' && github.event_name != 'schedule'
```

Comment:
Is this only for testing? How does it run if it's testing `!= 'schedule'`?

Reply:
The job will be triggered by either pull_request or schedule events.
This job will only be triggered by updating paths under pull_requests, which here are 
```
    - '.github/workflows/generate_issues.yml'
    - '.github/actions/testing_report_generation**'
```

Reply:
Then shouldn't it be `github.event_name == 'schedule'`?

Reply:
It is testing the github action tool and generate a new issue. It is mimicking the scheduled job, so it is not testing `!= 'schedule'`